app/controllers/targets_controller.rb

```diff
@@ -36,4 +50,8 @@ def target_params
   def topics
     @topics ||= Target::TOPICS
   end
+
+  def target
+    @target ||= current_user.targets.find(params[:id])
```

Comment:
I would do `find!` so that it returns not found in case someone tries to act over a target that does not exist or is not theirs. Opinion? @oscarsiniscalchi